sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 1.0.0-preview.6 - 2019-10-22
+
+- Renamed `DeviceCodeDetails` to `DeviceCodeInfo` and improved casing of the fields in the `ErrorResponse` type ([PR #5662](https://github.com/Azure/azure-sdk-for-js/pull/5662))
+- Improved the constructor signatures for `AuthorizationCodeCredential`, `DeviceCodeCredential`, `InteractiveBrowserCredential` and `managedIdentityCredential` so that it's clearer which parameters are optional and what additional values they accept ([PR #5668](https://github.com/Azure/azure-sdk-for-js/pull/5668))
```

Comment:
Not: capitalize the M in managedIdentityCredential

Reply:
Le oops